client/angular-marketplace/README.md

```diff
@@ -23,6 +23,34 @@ cd client/angular-marketplace
 npm install
 ```
 
+## Plugins
+We use for this project these plugins in VSCODE to help us with the development:
+- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+- [Angular Language Service](https://marketplace.visualstudio.com/items?itemName=Angular.ng-template)
+- [Angular Snippets](https://marketplace.visualstudio.com/items?itemName=johnpapa.Angular2)
+- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+
+
+## Code Linting Rules
+In this project is provided the .prettierrc file which has the basic configuration for Prettier. In the case of not being present, these are the settings:
+
+```JSON
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 300,
+  "html.format.wrapLineLength": 300,
+  "html.format.wrapAttributes": "auto"
+}
+```
+
+The file .prettierrc is placed outside the /src folder, we can format all the .ts files by running:
+```BASH
+npm run prettier-format
```

Comment:
I will propose to automatize all of this in a docker file later. The idea would be that in the root we have a docker-compose that use this folder to run the angular container (not a image), and have this automatized. Once the Angular installation was finished, I can help with that

Reply:
Yeah, we can do that later! I'll write it down.